test/python/basicaer/test_basicaer_integration.py

```diff
@@ -33,7 +32,7 @@ def setUp(self):
         self._qc2 = QuantumCircuit(qr, cr, name="qc2")
         self._qc1.measure(qr[0], cr[0])
         self.backend = BasicAer.get_backend("qasm_simulator")
-        self._result1 = execute(self._qc1, self.backend).result()
+        self._result1 = self.backend.run(self._qc1).result()
```

Comment:
Not that I imagine it will matter here, but technically this is a behavioural change due to the missing `transpile`.  Same comment for most changes in the next two files.